test/boulder-tools/Dockerfile

```diff
@@ -12,3 +12,5 @@ COPY requirements.txt /tmp/requirements.txt
 COPY boulder.rsyslog.conf /etc/rsyslog.d/
 COPY build.sh /tmp/build.sh
 RUN /tmp/build.sh
+RUN sed -i '/$ActionFileDefaultTemplate/s/^/#/' /etc/rsyslog.conf
+RUN sed -i '/imklog/s/^/#/' /etc/rsyslog.conf
```

Comment:
nit: I'd move these up to immediately after line 12, just so all the rsyslog stuff is together.

Reply:
Can't, they don't exist until rsyslog is installed on the previous line.